libs/wire-api/src/Wire/API/Conversation.hs

```diff
@@ -746,7 +746,7 @@ modelTeamInfo = Doc.defineModel "TeamInfo" $ do
   Doc.property "teamid" Doc.bytes' $
     Doc.description "Team ID"
   Doc.property "managed" Doc.bool' $
-    Doc.description "Is this a managed team conversation?"
+    Doc.description "(Not parsed any more) Is this a managed team conversation?"
```

Comment:
I thought we were changing this description as well?

Reply:
> I thought we were changing this description as well?

Sure, let me update this one as well. I'm not sure the model is used anywhere, but I'll update it anyway.